hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/InconsistentAmazonS3Client.java

```diff
@@ -171,14 +209,15 @@ public static InconsistentAmazonS3Client castFrom(AmazonS3 c) throws
   }
 
   @Override
-  public DeleteObjectsResult deleteObjects(DeleteObjectsRequest
-      deleteObjectsRequest)
+  public DeleteObjectsResult deleteObjects(
+      DeleteObjectsRequest deleteObjectsRequest)
       throws AmazonClientException, AmazonServiceException {
     for (DeleteObjectsRequest.KeyVersion keyVersion :
         deleteObjectsRequest.getKeys()) {
       registerDeleteObject(keyVersion.getKey(), deleteObjectsRequest
-          .getBucketName());
+        .getBucketName());
     }
+    maybeFail();
```

Comment:
I'd move the failure point above the for loop.

Reply:
done